lib/ruby_lsp/ruby_lsp_rails/addon.rb

```diff
@@ -47,6 +47,8 @@ def deactivate
         ).void
       end
       def create_code_lens_listener(response_builder, uri, dispatcher)
+        return unless T.must(@global_state).test_library == "rails"
```

Comment:
I decided to skip the entire listener because it's only used for test code lends atm.

Reply:
I may be worth creating a separate listener for that feature now that we have conditions like this.